File: server/src/main/java/com/uiptv/model/SeriesWatchState.java
package com.uiptv.model;

import com.uiptv.shared.BaseJson;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@EqualsAndHashCode(callSuper = false)
public class SeriesWatchState extends BaseJson {
    private String dbId;
    private String accountId;
    private String mode;
    private String categoryId;
    private String seriesId;
    private String episodeId;
    private String episodeName;
    private String season;
    private int episodeNum;
    private long updatedAt;
    private String source;
    private String seriesCategorySnapshot;
    private String seriesChannelSnapshot;
    private String seriesEpisodeSnapshot;
}

File: server/src/main/kotlin/com/uiptv/model/SeriesWatchState.kt
package com.uiptv.model

import com.uiptv.shared.BaseJson

data class SeriesWatchState @JvmOverloads constructor(
    var dbId: String? = null,
    var accountId: String? = null,
    var mode: String? = null,
    var categoryId: String? = null,
    var seriesId: String? = null,
    var episodeId: String? = null,
    var episodeName: String? = null,
    var season: String? = null,
    var episodeNum: Int = 0,
    var updatedAt: Long = 0,
    var source: String? = null,
    var seriesCategorySnapshot: String? = null,
    var seriesChannelSnapshot: String? = null,
    var seriesEpisodeSnapshot: String? = null
) : BaseJson()
